autoload/neomake/makers/ft/d.vim

```diff
@@ -49,8 +49,7 @@ function! neomake#makers#ft#d#dmd()
     return {
         \ 'args': l:args,
         \ 'errorformat':
-        \     '%-G%f:%s:,%f(%l): %m,' .
-        \     '%f:%l: %m',
+        \     '%f(%l\,%c): %m,'
```

Comment:
Changes look good to me. If not 100% sure you're done, but if you're comfortable with it I'll merge.